apps/consumer-client/pages/examples/add-pcd.tsx

```diff
@@ -62,11 +74,201 @@ export default function Page() {
         <button onClick={addWebAuthnPCD}>
           add a new webauthn credential to the passport
         </button>
+        <br />
+        <br />
+        <AddEthAddrPCDButton />
+        <br />
+        <br />
+        <AddEthGroupPCDButton />
       </ExampleContainer>
     </div>
   );
 }
 
+function AddEthAddrPCDButton() {
```

Comment:
This add eth addr pcd button is tangentially related to the PR. Just backfilled it.